# Tracing the origin of SARS-CoV-2 Omicron-like Spike sequences detected in wastewater 

Github repository to accompany the above manuscript. Link and scripts will be provided as they become available.

Martin M Shafer%,  Max J Bobholz%,  William C Vuyk%,  Devon A Gregory%,  Adelaide Roguet,  Luis A Haddock Soto, Clayton Rushford,  Kayley H Janssen,  Hunter J Ries,  Hannah E Pilch,  Paige A Mullen,  Rebecca B Fahney,  Wanting Wei, Matthew Lambert, Jeff Wenzel,  Peter Halfmann,  Yoshihiro Kawaoka,  Nancy A Wilson,  Thomas C Friedrich,  Ian W Pray,  Ryan Westergaard,  David H O'Connor, Marc Johnson^

%equal contribution

[View preprint](https://www.medrxiv.org/content/10.1101/2022.10.28.22281553v4)


^Corresponding Author  
Marc C. Johnson  
Phone: 573-882-1519  
marcjohnson@missouri.edu  
471c Bond Life Sciences Center  
1201 Rollins St  
Columbia, MO USA 65211



These sequence reads were generated by the Wisconsin State Laboratory of Hygiene using an Illumina MiSeq instrument  from 425 amplicons covering the whole SARS-CoV-2 genome amplified from wastewater samples with the QIAseq DIRECT SARS-CoV-2 Kit A. 

Illumina_WGS fastq files contain these unaligned sequence reads. Each sample should have both an R1 and and R2 fastq file, which result from Illumina's paired-end sequencing technique. Illumina reads are highly accurate and short.



